The current master is failing in CI

https://travis-ci.org/sweetalert2/sweetalert2-react-content/builds/390371045

```
./node_modules/.bin/tsc src/sweetalert2-react-content.d.ts --lib dom,es6 

src/sweetalert2-react-content.d.ts:56:37 - error TS2344: Type 'keyof T' does not satisfy the constraint 'string'.
  Type 'string | number | symbol' is not assignable to type 'string'.
    Type 'number' is not assignable to type 'string'.

56 type Overwrite<T, U> = Pick<T, Diff<keyof T, keyof U>> & U;

```

@toverux would you mind to take a look on this issue?

--- a/src/sweetalert2-react-content.d.ts
+++ b/src/sweetalert2-react-content.d.ts
@@ -41,16 +41,8 @@ interface ReactOptions {
   footer?: ReactElementOr<'footer'>;
 }
 
-// Diff<> and Owerwrite<> types below are inspired from this GitHub comment:
-// https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-307871458
-
-// Diff<T, U> is a subtraction operator for string literal types.
-// It returns the string types of T that are not in U.
-// ex: Diff<('a' | 'b' | 'c'), ('c' | 'd')>;  // 'a' | 'b'
-type Diff<T extends string, U extends string> =
-  ({ [P in T]: P } & { [P in U]: never } & { [x: string]: never })[T];
-
 // Overwrite<T, U> will take the properties of U and add to it the properties of T that are not in U.
 // This emulates an overwrite (override) of chosen properties of T with properties of U.
 // It works like { ...T, ...U } in JS.
-type Overwrite<T, U> = Pick<T, Diff<keyof T, keyof U>> & U;
+// https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-393936055
+type Overwrite<T, U> = Pick<T, Exclude<keyof T, keyof U>> & U;

